instrumentation-api-annotation-support/build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.jacoco-conventions")
+  id("otel.publish-conventions")
+}
+
+group = "io.opentelemetry.instrumentation"
+
+dependencies {
+  implementation(project(":instrumentation-api"))
+  implementation(project(":instrumentation-api-caching"))
```

Comment:
You don't need to depend on the caching module anymore, it's now built in the `instrumentation-api` module